cmd/utils.go

```diff
@@ -85,13 +86,28 @@ func ParseAddressToNative(ctx context.Context, addr string) (address.Address, er
 }
 
 func ParseAddressToEVM(ctx context.Context, addr string) (common.Address, error) {
-	lapi, closer, err := PoolsSDK.Extern().ConnectLotusClient()
+	if strings.HasPrefix(addr, "0x") {
+		return common.HexToAddress(addr), nil
+	}
+
+	re := regexp.MustCompile(`^[tf][0-9]`)
+	if re.MatchString(addr) {
+
+		lapi, closer, err := PoolsSDK.Extern().ConnectLotusClient()
+		if err != nil {
+			return common.Address{}, err
+		}
+		defer closer()
+
+		return parseAddress(ctx, addr, lapi)
+	}
+
+	as := util.AccountsStore()
+	evmAddr, _, err := as.GetAddrs(addr)
```

Comment:
Why did this logic get added here?

Reply:
This function was already accepting either an eth or a fil address, and if it was a fil address, it was going a lookup for the id address using Lotus.

This was used by a lot of the CLI commands that accept addresses, and I needed to add in the ability to also accept account names (eg. 'default', 'owner', 'operator').

The name of the function should probably be renamed because it now accepts both account names and addresses.

Reply:
I renamed it to `AddressOrAccountNameToEVM()` ... a bit wordy, but I'm bad at naming things.

Reply:
Got it! That's cool - so you should be able to transfer or withdraw to named accounts as well